hacking/aws_config/testing-iam-policy.json.j2

```diff
@@ -179,9 +179,9 @@
                 "rds:CreateDBInstanceReadReplica"
             ],
             "Resource": [
-                "arn:aws:rds:{{aws_region}}:{{aws_account}}:snapshot:snapshot-*",
-                "arn:aws:rds:{{aws_region}}:{{aws_account}}:snapshot:rds-*",
-                "arn:aws:rds:{{aws_region}}:{{aws_account}}:db:rds-*"
+                "arn:aws:rds:{{aws_region}}:{{aws_account}}:snapshot:ansible-test*",
```

Comment:
This isn't your doing, but I'm somewhat sure these IAM policies don't work as expected because they use partial wildcards http://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html